cypress/integration/rendering/classDiagram-v2.spec.js

```diff
@@ -433,11 +432,12 @@ describe('Class diagram V2', () => {
       { logLevel: 1, flowchart: { htmlLabels: false } }
     );
   });
-  it('17b: should handle the direction statement with RL', () => {
+
+  it('18a: should handle the direction statement with LR', () => {
```

Comment:
I have no idea why those tests are numbered either.

Reply:
> I have no idea why those tests are numbered either.

Maybe it's to make the image snapshot filenames sortable in alphabetical order?

But if it is, it's currently broken, since the files will be ordered: 1, 11, 12, 13, 2, 3, 4, ... Oh well, maybe it's something to fix in a future PR :shrug:.

Reply:
Let's remove numbers from new tests. 
Removing numbers from Old tests might mess up applitools, so we can do that as a separate PR.

Reply:
Please confirm this is resolved, as we will remove numbered tests later